api/task_lifecycle_client.go

```diff
@@ -56,9 +59,10 @@ func (c *TaskLifecycleClient) Scheme() string {
 	return c.url.Scheme
 }
 
-// CreateTask takes a task request and run option and sends this information to the client. It then returns
+// CreateTaskSimple takes a task request and run option and sends this information to the client. It then returns
```

Comment:
I made this change so as to not override the interface, this is useful for mocking down the line.

Reply:
Could we brainstorm a different name? "Simple" to me doesn't really reflect the behavior and implies that this method is a subset of the full create functionality. 

Alternatively, do we have a different solution for mocking than implementing the ClientInterface? It feels like we're not using it properly if we have to rename this method to make it work.

Reply:
hmm I may have another idea I can run by you here